modules/engine/src/main/java/com/opengamma/strata/engine/marketdata/ScenarioCalculationEnvironment.java

```diff
@@ -56,6 +57,10 @@
 @BeanDefinition(builderScope = "private")
 public class ScenarioCalculationEnvironment implements ImmutableBean {
 
+  /** The market data values which are the same in every scenario. */
+  @PropertyDefinition(validate = "notNull")
+  private final CalculationEnvironment baseData;
```

Comment:
Shared data is what I was thinking too.
On 17 Jul 2015 1:22 pm, "Stephen Colebourne" <notifications@github.com>
wrote:

> In
> modules/engine/src/main/java/com/opengamma/strata/engine/marketdata/ScenarioCalculationEnvironment.java
> <https://github.com/OpenGamma/Strata/pull/388#discussion_r34884205>:
>
> > @@ -56,6 +57,10 @@
> >  @BeanDefinition(builderScope = "private")
> >  public class ScenarioCalculationEnvironment implements ImmutableBean {
> >
> > +  /** The market data values which are the same in every scenario. */
> > +  @PropertyDefinition(validate = "notNull")
> > +  private final CalculationEnvironment baseData;
>
> How about commonData or sharedData.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenGamma/Strata/pull/388/files#r34884205>.
>